site/sfguides/src/getting_started_with_traces/getting_started_with_traces.md

```diff
@@ -0,0 +1,445 @@
+author: Brad Culberson
+id: getting_started_with_traces
+summary: Getting Started with Traces
+categories: featured,getting-started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Snowpark Python, Notebooks, Observability
+
+# Getting Started with Traces
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
```

Comment:
Does the UI add units to these? (as in Duration: 1 minute)

Reply:
yes, it adds them up and shows total time and time remaining

Reply:
Great thanks, we can close this.